mcnp2cad.cpp

```diff
@@ -88,8 +88,7 @@ class GeometryContext {
   std::string materialName( int mat, double rho ){
     std::string ret;
     std::stringstream formatter;
-    if(Gopt.uwuw_names){
-      bool mass_density = false;
+       bool mass_density = false;
```

Comment:
The indentation seems inconsistent.

Also, I think the closing brace from this now-deleted if statement is still floating around somewhere?

Reply:
Ah, yeah that deletion was a mistake. Thanks! Just replaced that line and fixed the indentation.